engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2246,6 +2306,67 @@ public LogicalOperatorInfo getLogicalOperatorInfo(String operatorName)
     return fillLogicalOperatorInfo(operatorMeta);
   }
 
+  public LogicalModuleInfo getLogicalModuleInfo(String moduleName, boolean flatten, LogicalPlan dag, String ancestry)
```

Comment:
If there are multiple instances of a module, can user not request for information for particular module id? the way user can do for physical instance of an operator?